types/http-graceful-shutdown/http-graceful-shutdown-tests.ts

```diff
@@ -0,0 +1,21 @@
+import * as GracefulShutdown from 'http-graceful-shutdown';
```

Comment:
It's better to use `import...require` https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports

Reply:
I did see that - but notably, that specifically refers to the case where the namespace is empty, and I'm currently using it.

If I were to use an `export =` style export, would I still be able to give a name to the Options interface? As far as I know, that method only allows for a single export.

If you anticipate that this will break in the future (and based on that linked StackOverflow post, it might) I can easily make this change - but do you have any suggestions on what would be the best way to expose the Options type?

Reply:
How about just changing this line to `import GracefulShutdown = require('http-graceful-shutdown')`, I believe you can still get `GracefulShutdown.Options`

Reply:
Thanks - I misunderstood your question, I thought the namespace's existance at all was the problem. I'm happy to make that change, and have already sent it in.